app/src/main/java/edu/uc/hickmadc/customnotifications/adapters/GroupsAdapter.kt

```diff
@@ -23,23 +25,21 @@ class GroupsAdapter(private val dataSet: ArrayList<Group>) :
         )
     }
 
-    // Replace the contents of a view (invoked by the layout manager)
+    /**
+     * Replace the contents of a view (invoked by the layout manager)
+     * Gets a group from dataSet and bind it to the empty viewHolder
+     */
     override fun onBindViewHolder(holder: RecyclerView.ViewHolder, position: Int) {
-        // get group from dataSet and bind it to the empty viewHolder
         (holder as GroupViewHolder).bind(dataSet[position])
     }
 
-    // Return the size of your dataset (invoked by the layout manager)
+    /**
+     * Return the size of your dataset (invoked by the layout manager)
+     */
     override fun getItemCount() = dataSet.size
 
     class GroupViewHolder(private val binding: ListItemGroupBinding) :
         RecyclerView.ViewHolder(binding.root) {
-        init {
-            // Define click listener for the ViewHolder's View.
-//            binding.setClickListener {
-//                Logger.getAnonymousLogger().warning("Clicked group row")
-//            }
```

Comment:
Again, just delete commented out code. I know it can be useful for debugging but when an outsider looks at it, I'm confused.